siriushlafac/VERSION

```diff
@@ -1 +1 @@
-0.8.2
+0.8.3
```

Comment:
It seems the changes here bring a `0.9.0`

Reply:
done!